explorer/deploy/webAppCreator.ts

```diff
@@ -447,6 +465,12 @@ class WebsiteStep extends WebAppCreatorStepBase {
                 await vscode.window.showWarningMessage(nameAvailability.message);
             }
         }
+        try {
+            await this.acquireRegistryLoginCredentials();
+        } catch (error) {
+            //Admin was not enabled, cannot proceed
+            throw new Error(('Admin not enabled'));
```

Comment:
You're making an assumption about why this failed - it could fail for any number of reasons, including network out - this message could mislead users.

Reply:
You should simply not have the try/catch here - let the exception bubble up (sync up with my PR I mentioned above).